### PR TITLE
DEVDOCS-3856: Update custom-templates

### PR DESCRIPTION
**Pull request by traci707**
# [DEVDOCS-3856](https://bigcommercecloud.atlassian.net/browse/DEVDOCS-3856)

## What changed?
Updated text per ticket
See the screenshot of the test to confirm the change

### CODE REVIEW

File: docs/stencil-docs/storefront-customization/custom-templates.md [hunk 1]
```diff
@@ -118,7 +118,7 @@ Beyond the single URL mapped to each template in the above examples, you have th
 
 ## Specifying custom front matter
 
-You can specify front matter on a custom template if you only need to render certain resources (such as 'New Products') on that page. Specifying only the attributes you need will reduce page load time. If you don't explicitly specify front matter for your custom template, the front matter for the default page template will be available. See [Using Front Matter](/stencil-docs/storefront-customization/using-front-matter) for more information on using front matter.
+You can specify front matter on a custom template by injecting data into the front matter of a generic template first, then the custom template will inherit it. If you don't explicitly specify front matter for your custom template, the front matter for the default page template will be available. See [Using Front Matter](/stencil-docs/storefront-customization/using-front-matter) for more information on using front matter.
```
Comment (slsriehl) [inline docs/stencil-docs/storefront-customization/custom-templates.md:121]: how about this?

```suggestion
You can't specify front matter on a custom template, but you can still access front matter-injected data. Custom templates inherit front matter from their corresponding default template. Specify the data you'd like your custom template to access in the default template's front matter. See [Using Front Matter](/stencil-docs/storefront-customization/using-front-matter) for more information on using front matter.
```

Comment (slsriehl) [reply docs/stencil-docs/storefront-customization/custom-templates.md:121]: @hatertron3000 this is your ticket -- is this correct?

Comment (hatertron3000) [reply docs/stencil-docs/storefront-customization/custom-templates.md:121]: This is accurate. An example may be helpful (i.e. inject into templates/pages/brand.html to access the data in templates/pages/custom/brand/my-custom-brand.html).

File: docs/stencil-docs/storefront-customization/custom-templates.md [hunk 2]
```diff
@@ -118,7 +118,48 @@ Beyond the single URL mapped to each template in the above examples, you have th
...
+
+
+
+    
+
+    
+
+    
+
+<div>You have {{cart.items}} items in your cart!</div>
```
Comment (hatertron3000) [inline docs/stencil-docs/storefront-customization/custom-templates.md:155]: I gave a bad example from memory. it should be `{{cart.quantity}}`, not `{{cart.items}}` in this line.
@traci707

**Review by slsriehl — APPROVED** [review]
🥳

**Pull request merged**